molecule/default/create.yml

```diff
@@ -10,7 +10,7 @@
     - community.crypto
   vars:
     # Run config handling
-    default_run_id: "{{ lookup('password', '/dev/null chars=ascii_lowercase length=5') }}"
+    default_run_id: "{{ lookup('env', 'BUILD_NUMBER') }}"
```

Comment:
FIXME needs to be different for each job in the ec2 matrix otherwise it will fail